gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
```

Comment:
Suggestion (💡): How about taking this opportunity and introduce `distributionSha256Sum` here as well (just like it was done recently for the [pocket-casts-android](https://github.com/Automattic/pocket-casts-android) repo (see code [here](https://github.com/Automattic/pocket-casts-android/blob/main/gradle/wrapper/gradle-wrapper.properties#L7) and PR [here](https://github.com/Automattic/pocket-casts-android/pull/419)).

Thus, what you could do is:
1. Find the correct [complete (-all) zip checksum](https://gradle.org/release-checksums/) for this `7.5.1` version of Gradle.
2. Run the below command after having copy-pasted the correct checksum into the `gradle-distribution-sha256-sum` option:
 
`./gradlew wrapper --gradle-version=7.5.1 --distribution-type=all --gradle-distribution-sha256-sum=db9c8211ed63f61f60292c69e80d89196f9eb36665e369e7f00ac4cc841c2219`